src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -0,0 +1,256 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "lazy_tensor.hpp"
+
+using ov::npuw::weights::ConcatMeta;
+using ov::npuw::weights::ConstPtr;
+using ov::npuw::weights::LazyTensor;
+using ov::npuw::weights::LTData;
+using ov::npuw::weights::Transform;
+using ov::npuw::weights::TransformType;
+
+namespace ov {
+namespace npuw {
+namespace weights {
+
+struct LazyTensorImpl {
+public:
+    explicit LazyTensorImpl() = default;
+    explicit LazyTensorImpl(const TransformType& type, const Transform& transform);
```

Comment:
there's no need to be explicit for a two-argument constructor.

Reply:
Done